packages/dev/core/src/Meshes/mesh.ts

```diff
@@ -2054,13 +2054,17 @@ export class Mesh extends AbstractMesh implements IGetSetVerticesData {
 
         const scene = this.getScene();
         const engine = scene.getEngine();
+        const material = subMesh.getMaterial();

```

Comment:
I am not sure we should add this here as it is inside of the hot path. We should also get the info on the _draw callers ? cc @deltakosh